channel/src/main/java/com/synopsys/integration/alert/channel/jira/cloud/JiraCloudChannel.java

```diff
@@ -55,8 +57,9 @@ public JiraCloudChannel(JiraCloudChannelKey jiraChannelKey, Gson gson, AuditAcce
 
     @Override
     protected IssueTrackerContext getIssueTrackerContext(DistributionEvent event) {
-        FieldUtility fieldUtility = event.getFieldUtility();
-        return jiraCloudContextBuilder.build(fieldUtility);
+        ConfigurationModel globalConfig = event.getChannelGlobalConfig()
+                                              .orElseThrow(() -> new AlertRuntimeException(new AlertConfigurationException("Missing Jira Cloud global configuration")));
```

Comment:
Instead of nesting exceptions, we should make a runtime configuration exception.

Reply:
Why not just throw the AlertRuntimeException with the error message?  The nested exception doesn't buy you anything.

Reply:
I created a Jira issue to resolve this. One reason for wrapping it is because the `AlertConfigurationException` is the correct exception to throw, so when someone comes to resolve this, they will have the correct alternative available.